src/components/app/details/appOverview/AppOverview.tsx

```diff
@@ -311,3 +303,11 @@ export default function AppOverview({ appMetaInfo, getAppMetaInfoRes }: AppOverv
         </div>
     )
 }
+
+export const renderDeployedTime = (_env, isAgroInstalled) => {
+    if (_env.lastDeployed) {
+        return handleUTCTime(_env.lastDeployed, true)
+    } else {
+        return isAgroInstalled ? '' : 'Not deployed'
+    }
+}
```

Comment:
Please move to either time utils or some app common component. As it's returning the string.